modules/dataframe/include/inviwo/dataframe/io/csvreader.h

```diff
@@ -89,6 +89,20 @@ class IVW_MODULE_DATAFRAME_API CSVReader : public DataReaderType<DataFrame> {
     CSVReader& setNumberOfExampleRows(size_t rows);
     size_t getNumberOfExamplesRows() const;
 
+    /** 
+    * sets the string that indicates that a row should be commented/removed
```

Comment:
from [ResearchDaniel](https://github.com/ResearchDaniel)
Suggestion:
Set characters indicating a line comment, for example "//". Lines starting with \p comment will be ignored during parsing. By default it is set to empty, meaning that no lines will be treated as comments.